src/landpatterns/dual-row.stanza

```diff
@@ -291,3 +273,95 @@ public defn build-smd-pin-1 (
   add-artwork(vp, Silkscreen("pin-1-marker", side), Circle(marker-pos, line-width), name = "pin-1-dot", class = "pin1-marker")
 
 
+doc: \<DOC>
+Dual-Row Grid Planner
```

Comment:
Probably lack of EE context on my side, but wouldn't `Dual-Column` be more intuitive here? This also explains my observations when implementing the terminal blocks...

Reply:
`Dual Row` connectors is a pretty common term of art. 

Try google [dual row connector](https://www.google.com/search?q=dual+row+connector
) vs [dual column connector](https://www.google.com/search?q=dual+column+connector). There is a night and day difference.